### ISSUE TITLE
Create FE Concept

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#18]
## Userstory
* POs create FE concept
* FE SDs create FE concept
* Presentation and discussion

## Acceptance criteria
* Agreement regarding design and project requirements

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by op-hub** [comment]
Small requirement notes
https://docs.google.com/document/d/1Y8mXNuit3eA8AI3zsE1gk8xR9Givt_Uvndifn5rclHE/edit#heading=h.nyheprkbuwq0

**Comment by Lucas-Nan** [comment]
was discussed together with PO Oli and FE team in meeting friday 03.05.

**Comment by Lucas-Nan** [comment]
moved back because "review" is supposed to be officially done ath the sprint end

**Comment by op-hub** [comment]
Short summary of our meeting: https://docs.google.com/document/d/1Hm3zwF_q7XPSMzWqKifDTmNBJLSGf5O-oW060w_o4g0/edit#heading=h.xs5kisbpgtln
